dom/nodes/Document-constructor.html

```diff
@@ -47,7 +48,7 @@
 
 test(function() {
   var doc = new Document();
-  var a = doc.createElement("a");
+  var a = doc.createElementNS("http://www.w3.org/1999/html", "a");
```

Comment:
Wrong namespace, missing "x", causes http://w3c-test.org/submissions/10712/dom/nodes/Document-constructor.html to fail. (Discussed in IRC but recording for posterity.)

It may help this test to include

```js
assert_equals(a.constructor, HTMLAnchorElement, "Sanity check: the created element is a HTMLAnchorElement");
```

to make the failure mode clearer in this area.

Reply:
Done!